Resources/doc/index.md

```diff
@@ -31,9 +31,9 @@ public function registerBundles()
 Generate the SSH keys :
 
 ``` bash
-$ mkdir -p app/var/jwt
-$ openssl genrsa -out app/var/jwt/private.pem -aes256 4096
-$ openssl rsa -pubout -in app/var/jwt/private.pem -out app/var/jwt/public.pem
+$ mkdir -p var/jwt # Symfony2
```

Comment:
Not sure to understand this comment, as the example doesn't work anymore for Symfony2?